test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -885,6 +888,72 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
       });
     });
 
+    Expect.describe('Typing biometrics', function () {
+      itp('does not contain typing pattern header in primary auth request if feature is disabled', function () {
+        return setup({ features: { trackTypingPattern: false }})
+        .then(function (test) {
+          $.ajax.calls.reset();
+          test.form.setUsername('testuser');
+          test.form.setPassword('pass');
+          test.setNextResponse(resSuccess);
+          spyOn(TypingUtil, 'track').and.callFake(function (target) {
+            expect(target).toBe('okta-signin-username');
+          });
+          test.form.submit();
+          return tick();
```

Comment:
can we replace this with a different wait? maybe `return Expect.waitForSpyCall(test.successSpy)`
(trying to avoid `tick`s if possible since ticks are not really deterministic)